tvb_documentation/tutorials/tutorial_1_BuildingYourOwnBrainNetworkModel.rst

```diff
@@ -297,39 +303,33 @@ greater than zero producing coloured noise.
 
 
 1. After configuring a model similar to the one presented in
-   *AnatomyOfARegionSimulation\_b*, we select **HeunStochastic** as our
+   *AnatomyOfARegionSimulation\_b*, we select **Stochastic Heun** as our
    integration scheme.
 
-2. Set the values for :math:`\boldsymbol{\tau=0}` and **seed=42**.
+2. Set the values for :math:`\boldsymbol{\tau=0}` and **noise_seed=42**.
 
 3. Set the noise dispersion, :math:`\mathbf{D=0.005}`
 
+.. figure:: figures/BuildingYourOwnBrainNetworkModel_IntegratorSettings.png
+   :scale: 30%
+
 *AnatomyOfARegionSimulation\_b* and
 *AnatomyOfARegionSimulation\_stochastic* have the same parameters but
 the latter has an extra background noisy input.
 
-Observe the differences using the *Spectrogram of the Wavelet Transform*.
```

Comment:
Have you managed to generate these?

Reply:
I have a suspicion that no matter what kind of simulation we configure, the Wavelet will look the same way or at least very similar. I am not sure though. I generated the Wavelet for this simulation in the 1.5.8 version. That one looks somewhat similar to the one from the tutorial, but I still don't think it is similar enough.